lib/linguist/languages.yml

```diff
@@ -2668,6 +2669,16 @@ OpenSCAD:
   tm_scope: none
   ace_mode: scad
 
+OpenType Font Feature:
+  type: programming
```

Comment:
Well, maybe it’s `markup` (like Postscript and CSS) but OT features are used to implement substitution algorithms etc. That sounds like programming to me. It’s certainly not just `data` like an image or config file.